apis/openstack-swift/src/main/java/org/jclouds/openstack/swift/v1/blobstore/RegionScopedSwiftBlobStore.java

```diff
@@ -679,7 +679,7 @@ public void downloadBlob(String container, String name, File destination, Execut
 
       ListeningExecutorService listeningExecutor = MoreExecutors.listeningDecorator(executor);
       RandomAccessFile raf = null;
-      File tempFile = new File(destination.getName() + "." + UUID.randomUUID());
+      File tempFile = new File(destination + "." + UUID.randomUUID());
```

Comment:
> With this change jclouds will create the temporary file in the same directory as the user specified.

Clear - thanks. My question was more: do we know that `destination` here will indeed be an absolute path?